crypto/x509/x509_set.c

```diff
@@ -222,6 +222,18 @@ static void x509_sig_info_init(X509_SIG_INFO *siginf, const X509_ALGOR *alg,
         return;
     /* Security bits: half number of bits in digest */
     siginf->secbits = EVP_MD_size(md) * 4;
+    /* SHA1 and MD5 are known broken. Reduce security bits so that they're no
```

Comment:
nit "/*" on a line by itself.  I wouldn't say SHA1 and MD5 are broken to the same level, but I don't get into that argument :)

Reply:
are known _to be_ broken